Plots/WRF/NCL_dataonmap_10.py

```diff
@@ -0,0 +1,93 @@
+"""
+NCL_datagrid_4_lg.py
+==================
+This script illustrates the following concepts:
+    - Plotting WRF data on native grid
+    - Plotting data using wrf python functions
+    - Following best practices when choosing a colormap
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/dataonmap_10.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/dataonmap_10_1_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+from netCDF4 import Dataset
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+import cartopy.crs as ccrs
+from wrf import (getvar, to_np, latlon_coords, get_cartopy)
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in the data
+
+wrfin = Dataset(gdf.get("netcdf_files/wrfout_d01_2003-07-15_00:00:00"), decode_times=True)
+
+q2 = getvar(wrfin, "Q2")
+
+###############################################################################
+# Plot the data
+
+lats, lons = latlon_coords(q2)
+
+# The `get_cartopy` wrf function will automatically find and use the 
+# intended map projection for this dataset 
+fig = plt.figure(figsize=(10,10))
+cart_proj = get_cartopy(q2)
+ax = plt.axes(projection=cart_proj)
+
+# Add filled contours
+plt.contourf(to_np(lons),
```

Comment:
Could we include something here or in the top docstring linking back to the color examples/tutorials explaining best practices?

Reply:
I just added a line in the "concepts illustrated" portion which links to one of the colormap examples